src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -238,16 +239,17 @@ internal sealed partial class ConsoleHost
                         throw hostException;
                     }
 
+                    ProfileOptimization.StartProfile(
+                        s_theConsoleHost.LoadPSReadline()
+                            ? "StartupProfileData-Interactive"
+                            : "StartupProfileData-NonInteractive");
```

Comment:
We use the strings in some places. Make sense to define consts?

Reply:
Those strings are only used once here, so I wouldn't bother to define consts for them.